qt-models/divetripmodel.cpp

```diff
@@ -389,26 +389,18 @@ QString DiveItem::displaySac() const
 {
 	QString str;
 	struct dive *dive = get_dive_by_uniq_id(diveId);
-	if (dive->sac) {
-		const char *unit;
-		int decimal;
-		double value = get_volume_units(dive->sac, &decimal, &unit);
-		return QString::number(value, 'f', decimal);
-	}
-	return QString("");
+	if (!dive->sac)
+		return str;
```

Comment:
I think it would be better to either use `return {};` or (if you prefer pre-C++11) `return QString();` and remove the `QString str;`-line.

There is no point in constructing a `QString` object in the normal case.